spec/manageiq_performance/configuration_spec.rb

```diff
@@ -84,92 +157,7 @@
   end
 
   describe "default config" do
-    it "defines ManageIQPerformance.config.default_dir" do
-      expect(ManageIQPerformance.config.default_dir).to eq("tmp/manageiq_performance")
-      expect(ManageIQPerformance.config["default_dir"]).to eq("tmp/manageiq_performance")
```

Comment:
Yeah, I agree, but the rational for doing this was that when I was testing the `with_config` stuff, I wanted to basically re-run all of these tests again, but minus the one that was changed.  So I sacrificed readability and complexity for DRY-ness to avoid doing a copy paste and doubling the size of the test.

Am on the fence about this, so any rational one way or the other would probably sway me on this.